src/commands/createFunctionApp/createCreateFunctionAppComponents.ts

```diff
@@ -58,7 +58,13 @@ export async function createCreateFunctionAppComponents(context: ICreateFunction
         wizardContext.hasDurableTaskScheduler = await durableUtils.getStorageTypeFromWorkspace(language, wizardContext.workspaceFolder.uri.fsPath) === DurableBackend.DTS;
     }
 
-    promptSteps.push(new SiteNameStep(context.dockerfilePath ? "containerizedFunctionApp" : "functionApp"));
+    // SiteNameStep pre-requisites
+    if (!context.advancedCreation) {
+        LocationListStep.addStep(wizardContext, promptSteps);
+    } else {
+        CustomLocationListStep.addStep(wizardContext, promptSteps);
+        promptSteps.push(new ResourceGroupListStep());
```

Comment:
One bad experience with resource group list step being here is that even it doesn't use my RG name for anything including the app name. I was kind of expecting an auto complete there.